RELEASE-PROCESS.md

```diff
@@ -0,0 +1,43 @@
+# Release Process
+
+The process of releasing a new version of the KEDA HTTP Addon involves a few steps, detailed below.
+
+>The process herein is largely automated but we recognize that there may be more that we can automate. If you find something that _can_ and _should_ be automated, and you believe that you know how, please [submit an issue](https://github.com/kedacore/http-add-on/issues/new?assignees=&labels=needs-discussion%2Cfeature-request&template=Feature_request.md) explaining how.
+
+## 0: Current and new versions
+
+Please go to the [releases page](https://github.com/kedacore/http-add-on/releases) and observe what the most recent release is. Specifically, note what the _tag_ of the release is. For example, if [version 0.1.0](https://github.com/kedacore/http-add-on/releases/tag/v0.1.0) is the latest release (it is as the time of this writing), the tag for that is `v0.1.0`.
+
+To determine the new version, follow [SemVer guidelines](https://semver.org). Most releases will increment the PATCH or MINOR version number.
+
+## 1: Submit a PR to the [Helm Charts Repository](https://github.com/kedacore/charts)
```

Comment:
I'd move this to step 2 since there is no new app version yet.

Worst case the container was shipped but the Chart PR is still in review.

Reply:
@arschles Automerge ignores comments, would you mind taking a look at this comment and share what you think?